mixer/pkg/runtime/handler/table.go

```diff
@@ -170,10 +171,20 @@ func (t *Table) Cleanup(current *Table) {
 			err = panicErr
 		}
 
-		if reportErr := entry.env.reportStrayWorkers(); reportErr != nil {
-			log.Warnf("unable to report if there are any stray go routines scheduled by the adapter '%s': %v",
-				entry.Name, reportErr)
-		}
+		go func(adapterEnv env, name string) {
+			strayWorkersFound := adapterEnv.hasStrayWorkers()
+			for i := 0; i < 10 && strayWorkersFound; i++ {
+				adapterEnv.Logger().Debugf("Found stray workers for adapter: %s; will check again in 10s.", name)
+				time.Sleep(1 * time.Second)
```

Comment:
Pull out this into a constant and match the text above (10s vs 1s)

Reply:
good point. pulled it out and made it more suitable for testing (and improved coverage).